### PR TITLE
Add docs templates and workflows

### PR DESCRIPTION
**Pull request by bsmth**
Adding template docs, see https://github.com/mdn/mdn/issues/323

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: "Issue report"
```
Comment (hamishwillee) [inline .github/ISSUE_TEMPLATE/bug.yml:1]: @bsmth Most bugs come from people following the tutorial on MDN incorrectly then raising bugs against this repo. That's why the original issue template had the information below. Is there some way we can add it back?

----

Before posting an issue, please confirm:
- [ ] Issue can be reproduced on THIS repository (setup using [quickstart instructions](https://github.com/mdn/django-locallibrary-tutorial#quick-start)).
- [ ] I am using a clean/fresh database
- [ ] I have provided output logs and information about my environment setup below.

Comment (bsmth) [reply .github/ISSUE_TEMPLATE/bug.yml:1]: >Is there some way we can add it back?

Of course, let me add it back 👍🏻

Comment (hamishwillee) [reply .github/ISSUE_TEMPLATE/bug.yml:1]: Thanks!

Comment (bsmth) [reply .github/ISSUE_TEMPLATE/bug.yml:1]: Added these checkboxes back in https://github.com/mdn/django-locallibrary-tutorial/pull/113/commits/63450e7b08b7a0f9eb3d6510535df06a79d74857

**Review by hamishwillee — APPROVED** [review]
Thanks for adding the checkbox items. I am not sure about the "right structure" for all of this so approving, but leaving to someone more expert to merge.

**Comment by bsmth** [comment]
Thanks, Hamish, it would be good to have @dipikabh take a last look

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    attributes:
+      label: "Before submitting the issue, check you have done the following:"
+      description: Make sure you have provided all the information below so we can help resolve your issue.
+      options:
+        - label: Issue can be reproduced on this repository following [quick-start instructions](https://github.com/mdn/django-locallibrary-tutorial#quick-start).
+          required: true
+        - label: I am using a clean/fresh database.
+          required: true
+        - label: I have included output logs and information about my environment setup below.
+          required: true
```
Comment (dipikabh) [inline .github/ISSUE_TEMPLATE/bug.yml:22]:
```suggestion
      label: "Before submitting the issue, confirm that you've checked everything in the following list, and mark them as completed here:"
      description: Make sure you have provided all the information below so we can help resolve your issue.
      options:
        - label: Issue is reproducible on this repository by following the [quick-start instructions](https://github.com/mdn/django-locallibrary-tutorial#quick-start).
          required: true
        - label: A clean/fresh database was used when the issue was encountered.
          required: true
        - label: Information about the environment setup when the issue was encountered is included in the issue.
          required: true
        - label: Output logs are included in the issue.
          required: true
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for these updates, @bsmth!

I've added some edits to the `bug.yml` file.
Question: how do we ensure that whatever was earlier pointing to the `ISSUE_TEMPLATE/django-locallibrary-issue-template.md` file is now pointing to `ISSUE_TEMPLATE/bug.yml`?

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 2 above)
Comment (bsmth) [reply .github/ISSUE_TEMPLATE/bug.yml]: Thanks 👍🏻

**Review by dipikabh — APPROVED** [review]
Thank you!

**Pull request merged**